src/Resources/dev/pages/payment/index.js

```diff
@@ -19,7 +19,8 @@ const Payment = {
       $(this.options.trigger).show();
     }
     $("input[id*=sylius_checkout_select_payment]").on("change", function () {
-      if ($(this).val() === "oney") {
+
+      if ($('label[for=' + $(this).attr('id') + ']').attr('data-gateway') === 'oney') {
```

Comment:
alternate version :
```js
if ($(`label[for=${$(this).attr('id')]}`).data("gateway") === "oney"
```

Reply:
Could you use suggestion feature ? :smile:

Reply:
Done :ok_hand: